keps/sig-autoscaling/20190307-configurable-scale-velocity-for-hpa.md

```diff
@@ -233,63 +250,121 @@ Example for `CurReplicas = 10` and HPA controller cycle once per a minute:
 
     recommendations = [9, 8, 9, 9, 8, 9, 8, 9, 8, 7]
 
-  The algorithm picks the largest value `9` and change the number of replicas `10 -> 9`
+  The algorithm picks the largest value `9` and changes the number of replicas `10 -> 9`
+
+#### Story 6: Avoid false positive signals for scaling up
+
+This mode is useful in Data Processing pipelines when the number of replicas depends on the number of events in the queue.
+The users want to scale up quickly if they have a high number of events in the queue. 
+However, they do not want to react to false positive signals, i.e. to short spikes of events.
+
+Create an HPA with the following behavior:
+
+```yaml
+behavior:
+  scaleUp:
+    stabilizationWindowSeconds: 300
```

Comment:
Why not `delay`?

Reply:
`delay` kind of implies it's a one time thing. `stabilizationWindow` would be more appropriate considering it's a moving window in which previous recommendations are considered to stabilize the scaling. I will explain this when I update the documentation for this feature.